ff/serving/provider/online_test.go

```diff
@@ -56,6 +61,8 @@ func TestOnlineStores(t *testing.T) {
 		{LocalOnline, []byte{}, false},
 		{RedisOnline, redisMockConfig.Serialized(), false},
 		{RedisOnline, redisLiveConfig.Serialized(), true},
+		{DynamoDBOnline, dynamoConfig.Serialized(), false},
```

Comment:
Only need true or false. Redis has both because we have a mock redis cluster and a live redis cluster we test on